docs/custom_metrics_hpa.md

```diff
@@ -0,0 +1,39 @@
+### Enabling custom metrics hpa for starboard exporter
+
+#### Required components
+
+Prometheus adapter
+
+#### Steps
+
+1. Install prometheus adapter.
+    1. Configure helm chart to connect with prometheus to get the metrics
+        ```
+        prometheus:
+          # Value is templated
+          url: http://prometheus-operated.gaia-prometheus.svc
+          port: 9090
+          path: "/gaia"
```

Comment:
Probably shouldn't use the `gaia` examples here. What would it be in a standard install?

Reply:
The standard on would be empty. Fixed it in the latest commit.